openmc/data/thermal.py

```diff
@@ -40,6 +40,28 @@
                   'zrzrh': 'c_Zr_in_ZrH', 'zr-h': 'c_Zr_in_ZrH', 'zr/h': 'c_Zr_in_ZrH'}
 
 
+def get_thermal_name(name):
+    """Get proper S(a,b) table name, e.g. 'HH2O' -> 'c_H_in_H2O'"""
+
+    if name in _THERMAL_NAMES.values():
+        return name
+    elif name.lower() in _THERMAL_NAMES:
+        return _THERMAL_NAMES[name.lower()]
+    else:
+        # Make an educated guess?? This actually works well for
+        # JEFF-3.2 which stupidly uses names like lw00.32t,
+        # lw01.32t, etc. for different temperatures
+        matches = get_close_matches(
+            name.lower(), _THERMAL_NAMES.keys(), cutoff=0.5)
+        if len(matches) > 0:
+            return _THERMAL_NAMES[matches[0]]
+        else:
+            # OK, we give up. Just use the ACE name.
+            return 'c_' + name
+
+    return name
```

Comment:
Doesn't look like this statement can ever be reached. Not sure if it's my fault or yours.